server/index.js

```diff
@@ -132,6 +132,8 @@ io.on('connection', (socket) => {
     })
 });
 
+app.get('/getRoom', (req, res) => res.send(getRooms()))
```

Comment:
Good thinking adding the `getRoom` route to validate whether a room exists!

Reply:
Or maybe we can change it to `getRooms` because that's what the function is getting as opposed to just one room?

Reply:
i think changing it to /rooms would be best!